src/DynamoCore/Configuration/IPathResolver.cs

```diff
@@ -158,6 +158,12 @@ public interface IPathManager
         string PreferenceFilePath { get; }
 
         /// <summary>
+        /// Full path to the python template py file. This setting file is specific 
+        /// to the current user.
+        /// </summary>
+        string PythonTemplateFilePath { get; }
```

Comment:
because you are changing the interface here you may need to update: https://github.com/DynamoDS/DynamoRevit/blob/Revit2017/src/DynamoRevit/RevitPathResolver.cs

in a related PR to avoid breaking the Revit build.